### PR TITLE
STM32MP1 Updates

### PR DESCRIPTION
**Pull request by EmbeddedAndroid**
Here is a series of patches @mike-scott and I have been using to improve the STM support in the LmP.

### CODE REVIEW

**Review by mike-scott — APPROVED** [review]

**Comment by mike-scott** [comment]
LGTM

File: meta-lmp-bsp/dynamic-layers/stm-st-stm32mp/recipes-support/flashlayouts-stm32mp1/flashlayouts-stm32mp1.bb [hunk 1]
```diff
@@ -6,14 +6,48 @@ LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MPL-2.0;md5=815ca599c9df247a0c7
...
+        -e 's/@@BOARD_OFFSET_FSBL2@@/${LMP_FLASHLAYOUT_BOARD_OFFSET_FSBL2}/' \
+        -e 's/@@BOARD_OFFSET_SSBL@@/${LMP_FLASHLAYOUT_BOARD_OFFSET_SSBL}/' \
+        -e 's/@@BOARD_OFFSET_ROOT@@/${LMP_FLASHLAYOUT_BOARD_OFFSET_ROOT}/' \
+        ${WORKDIR}/${LMP_FLASHLAYOUT_TEMPLATE} > ${WORKDIR}/${LMP_FLASHLAYOUT}
 }
 
-do_deploy() {
-    install -m 0644 ${WORKDIR}/${LMP_FLASHLAYOUT} ${DEPLOYDIR}
+# deploy a .tar.gz file of artifacts
+do_deploy_append() {
```
Comment (ricardosalveti) [inline meta-lmp-bsp/dynamic-layers/stm-st-stm32mp/recipes-support/flashlayouts-stm32mp1/flashlayouts-stm32mp1.bb:31]: No need to append if this is the only definition you have. Just do_deploy (as it was previously) should be fine.

**Comment by ricardosalveti** [comment]
Just one minor comment, otherwise LGTM

**Comment by EmbeddedAndroid** [comment]
New version pushed to address @rsalveti comment.

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**